pymongo/mongo_client.py

```diff
@@ -75,6 +75,8 @@
 
 
 class MongoClient(common.BaseObject):
+    """Class that provides a full featured client for a MongoDB
+    instance, a replica set, or a set of mongoses."""
```

Comment:
I was curious how other drivers document their MongoClient's. I like [Java's description](http://mongodb.github.io/mongo-java-driver/3.8/javadoc/com/mongodb/client/MongoClient.html) the best:
```
A client-side representation of a MongoDB cluster.

Instances can represent either a standalone MongoDB server, a replica
set, or a sharded cluster. Instances of this class are responsible for
maintaining up-to-date state of the cluster, and possibly cache
resources related to this, including background threads for monitoring,
and connection pools.
```
 What do you think of using the same description here?

Reply:
SGTM

Reply:
+1 I like this for the docstring!

Reply:
Nice description! More well thought out than the one I provided

@behackett thanks for finding this changelog, I glanced over the docs a few times but couldn't find the change itself. In case anyone else gets curious, here is the full issue [bpo-15582](https://bugs.python.org/issue15582)